### PR TITLE
HS-951 Adding Adapter Step without Adapter module

### PR DESCRIPTION
**Pull request by FrancescoBuc86**
https://opennms.atlassian.net/browse/HS-951

### CODE REVIEW

File: minion/minion-flows/flows-parser/src/main/java/org/opennms/horizon/minion/flows/parser/TelemetryRegistryImpl.java [hunk 1]
```diff
@@ -91,7 +102,7 @@ public FlowsListener getListener(ListenerConfig listenerConfig) {
         }
         for (var factory : listenerFactories) {
             if (factory.getClass().getName().contains(listenerConfig.getClassName())) {
```
Comment (freddy61025) [inline minion/minion-flows/flows-parser/src/main/java/org/opennms/horizon/minion/flows/parser/TelemetryRegistryImpl.java:104]: if you use full class path in config. Then you may need factory.getListenerClass().getName().equals(listenerConfig.getClassName()). 

Also same for parser.

**Review by freddy61025 — CHANGES_REQUESTED** [review]
there still have one test fail.

**Comment by FrancescoBuc86** [comment]
> there still have one test fail.

Yes saw that, I'll take care of it

**Review by freddy61025 — APPROVED** [review]
LGTM

**Pull request merged**